docs/user_guide.md

```diff
@@ -4,63 +4,146 @@ This is a pre-release version of the extension and is not intended for general u
-`diffix.mark_personal(namespace, table_name, aid_columns...)` procedure, for example:
+A user may have one of three security labels, `direct`, `anonymized_trusted`, and `anonymized_untrusted`. Users with security label `direct` have full access to the data: no anonymization takes place. The other two labels determine whether a user that is subject to anonymization is treated as _trusted_ or _untrusted_.
+
+* For trusted users, Diffix prevents _accidental_ release of personal data. So long as trusted users do not try to bypass Diffix anonymization, answers to queries are anonymous.
+* For untrusted users, Diffix prevents _intentional_ release of personal data. Even for users that are malicious and try to break Diffix anonymization, answer to queries are anonymous.
+
+Trusted users have fewer SQL restrictions than untrusted users, and therefore have better analytic utility.
+
+For example, the command to assign the security label `anonymized_untrusted` to the `public_access` is:
 
 ```SQL
-CALL diffix.mark_personal('public', 'my_table', 'id', 'last_name');
+SECURITY LABEL FOR pg_diffix ON ROLE public_access IS 'anonymized_untrusted';
 ```
 
-In order to label a table as `public`, use the `diffix.mark_public(namespace, table_name)` procedure.
+The value of the setting variable `pg_diffix.default_access_level` determines the default user class for unlabeled regular users (i.e. not superusers). It is set to `direct` by default.
+
+### Security labels for tables
+
+Tables may have one of two security labels, `public` or `personal`.
+
+* Tables labeled as `personal` are anonymized by the extension (for `anonymized_trusted` and `anonymized_untrusted` users, not `direct` users).
+* Tables labeled as `public` are not anonymized: all users have full access to these tables.
+
+The procedure `diffix.mark_public(namespace, table_name)` labels a table as `public`.
+
+Note that unlabeled tables can not be queried by `anonymized_trusted` and `anonymized_untrusted` users (unless the setting variable `pg_diffix.treat_unmarked_tables_as_public` is set to `true`).
+
+### Security labels for columns
+
+Each personal table has one or more _protected entities_ associated with it. A protected entity is normally a person or a something associated with a person (a device or an account), but it could also be other things such as a household, a store, or a company.
 
-Regular users can be marked with the anoymization labels `direct`, `publish_trusted` or `publish_untrusted`.
-The value of the custom variable `pg_diffix.default_access_level` determines the access level for unlabeled regular users.
+Each protected entity must have at least one column that contains the identifier of the protected entity. We refer to these columns generally as AID columns (Anonymization ID). See (How to select AID columns)[#how-to-select-aid-columns] for more guidance.
+
+NOTE: if AID columns are not correctly labeled, the extension may fail to anonymize appropriately.
```

Comment:
e.g. the "columns" section to me has a better narrative, with "security label" only in the title, NOTE and `mark_personal` part, which could also be easily un-security-labelled or left as they are now. But the gist of the section doesn't mention labels (good)